app/templates/components/-report/metrics-chart.hbs

```diff
@@ -1,14 +1,21 @@
 <div class="card-body">
   <div class="row">
-    <div class="col">
+    <div class="col col-md-2 col-lg-3">
```

Comment:
Instead of trying to manually set the row size can you simply
flex wrap these?

Reply:
I just changed this to be a separate row to prevent it from being weird in between the two buttons.